dsl/src/main/scala/com.crobox.clickhouse/dsl/AggregationFunctions.scala

```diff
@@ -82,10 +82,20 @@ object AggregateFunction {
     def max[V](tableColumn: TableColumn[V]) =
       Max(tableColumn)
 
-    def timeSeries(tableColumn: TableColumn[Long],
-                   interval: MultiInterval,
-                   dateColumn: Option[TableColumn[DateTime]] = None) =
-      TimeSeries(tableColumn, interval, dateColumn)
+    /**
+      * This function will push back the timestamp represented by tableColumn to the start of this interval,
+      * this happens deterministically.
+      *
+      * Meaning that as long as the duration is the same, your groups will be in the same from/to timestamps
+      *
+      * This is useful for aggregating results by periods of time (group by month, 2 months, days, etc.)
+      *
+      * @param tableColumn
+      * @param interval
+      * @return
+      */
+    def timeSeries(tableColumn: TableColumn[Long], interval: MultiInterval) =
```

Comment:
Can we add a segment in the readme about time grouping? 
And some examples with code usages and expected multi interval deterministic behavior?

Reply:
I will do that right away (add it to the wiki). If everything else looks ok please merge =x